packages/test/test-end-to-end-tests/src/test/detachedContainerTests.spec.ts

```diff
@@ -362,20 +363,21 @@ describeFullCompat("Detached Container", (getTestObjectProvider) => {
         };
         const defPromise = new Deferred<void>();
         const container = await loader.createDetachedContainer(provider.defaultCodeDetails);
-        (container.deltaManager as any).submit = (type, contents, batch, metadata) => {
-            assert.strictEqual(contents.contents.contents.content.address,
-                crcId, "Address should be consensus register collection");
-            assert.strictEqual(JSON.stringify(contents.contents.contents.content.contents),
-                JSON.stringify(op), "Op should be same");
-            defPromise.resolve();
-            return 0;
-        };
 
         // Get the root dataStore from the detached container.
         const response = await container.request({ url: "/" });
         const dataStore = response.value as ITestFluidObject;
         const testChannel1 = await dataStore.getSharedObject<ConsensusRegisterCollection<string>>(crcId);
 
+        dataStore.context.containerRuntime.on("op", (message) => {
```

Comment:
Is the ordering here important or is this a cosmetic change?

Reply:
I need access to container runtime, and the only way to get to it is to reorder code here.
Change in order does not make any difference, as no ops are issues while we are resolving data store